package.json

```diff
@@ -53,6 +53,7 @@
     "@hmcts/uk-bank-holidays": "^1.0.2",
     "accessible-autocomplete": "^2.0.4",
     "applicationinsights": "^2.7.3",
+    "axios": "^1.7.7",
```

Comment:
I like axios, but is adding a different http client necessary? Isn't there one already?

Reply:
yeah can just rp, will refactor it that way

Reply:
I would say use fetch but this app is still on node 18 and it's experimental on that